src/Appwrite/Auth/OAuth2/Zoho.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Appwrite\Auth\OAuth2;
+
+use Appwrite\Auth\OAuth2;
+
+// Reference Material
+// https://zoho.com/accounts/protocol/oauth.html
+
+class Zoho extends OAuth2
+{
+    /**
+     * @var string
+     */
+    private string $endpoint = 'https://accounts.zoho.com';
+
+    /**
+     * @var string
+     */
+    private string $authEndpoint = 'https://accounts.zoho.com/oauth/v2/auth';
```

Comment:
This is used in only one place, can we remove this for now?

Reply:
Done